tests/integration/components/respond-to-resize-test.js

```diff
@@ -1,43 +1,47 @@
 import { moduleForComponent, test } from 'ember-qunit';
-import Ember from 'ember';
 import hbs from 'htmlbars-inline-precompile';
+import { triggerEvent } from 'ember-native-dom-helpers';
 
-moduleForComponent('respond-to-resize', {
+moduleForComponent('respond-to-resize', 'Integration | Component | respond to resize', {
   integration: true,
-  beforeEach: function () {
+  beforeEach: function() {
     window.requestAnimationFrame = setTimeout;
```

Comment:
I don't think this line is necessary anymore as the tests on CI are run on Chrome and not on PhantomJS. So `window.requestAnimationFrame` is correctly defined.